src/core/utils/otns.cpp

```diff
@@ -130,6 +130,31 @@ void Otns::EmitNeighborChange(otNeighborTableEvent aEvent, Neighbor &aNeighbor)
     }
 }
 
+void Otns::EmitTransmit(const Mac::TxFrame &aFrame)
+{
+    Mac::Address dst;
+    IgnoreError(aFrame.GetDstAddr(dst));
+    uint16_t frameControlField = aFrame.GetFrameControlField();
+    uint8_t  channel           = aFrame.GetChannel();
+    uint8_t  sequence          = aFrame.GetSequence();
```

Comment:
```suggestion
    Mac::Address dst;
    uint16_t frameControlField = aFrame.GetFrameControlField();
    uint8_t  channel           = aFrame.GetChannel();
    uint8_t  sequence          = aFrame.GetSequence();
    
    IgnoreError(aFrame.GetDstAddr(dst));
```
put variable declarations before any statement.